### DISCUSSION TITLE
FlareSolver on Synology

### DISCUSSION THREAD

**Discussion by Wells83** [FlareSolverr/FlareSolverr#36] [Q&A]
Hi, i have sonarr and jacket on my DSM 6 (synology nas)

How i can install your solution in this environment ?

**Comment by ngosang** [comment]
We provide Docker images for FlareSolverr. If your NAS supports it this is the way.
https://github.com/FlareSolverr/FlareSolverr#docker

If you are having connection problems, check the Docker port. After adding the docker it auto-assigned the `Local Port` as `Auto`.  Changing that to `8191` to match the `Container Port` fixes it. You can change it in : `Container -> Edit -> Port Settings -> Local Port`

If you can install it, add the instructions in this discussion. It can be useful for other users.
